From 355285726357c13c302c5669b99d6e8d238723f7 Mon Sep 17 00:00:00 2001
From: Martin Perina <mperina@redhat.com>
Date: Mon, 14 Apr 2014 08:32:15 +0200
Subject: [RHEL7 PATCH 3/7] Rename is_fence_kdump to is_pcs_fence_kdump

Renames is_fence_kdump to is_pcs_fence_kdump to identify that this
method should be used to detect fence_kdump configuration only in
Pacemaker clusters.

Related: rhbz#1086988
Signed-off-by: Martin Perina <mperina@redhat.com>
---
 dracut-module-setup.sh | 4 ++--
 kdump-lib.sh           | 4 ++--
 kdumpctl               | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/dracut-module-setup.sh b/dracut-module-setup.sh
index 0ce3885..3df71b1 100755
--- a/dracut-module-setup.sh
+++ b/dracut-module-setup.sh
@@ -20,7 +20,7 @@ depends() {
         _dep="$_dep drm"
     fi
 
-    if is_fence_kdump; then
+    if is_pcs_fence_kdump; then
         _dep="$_dep network"
     fi
 
@@ -421,7 +421,7 @@ kdump_check_iscsi_targets () {
 # also preserve '[node list]' for 2nd kernel /etc/fence_kdump_nodes
 kdump_check_fence_kdump () {
     local nodes
-    is_fence_kdump || return 1
+    is_pcs_fence_kdump || return 1
 
     # get cluster nodes from cluster cib, get interface and ip address
     nodelist=`pcs cluster cib | xmllint --xpath "/cib/status/node_state/@uname" -`
diff --git a/kdump-lib.sh b/kdump-lib.sh
index 8de86b6..7103ba9 100755
--- a/kdump-lib.sh
+++ b/kdump-lib.sh
@@ -27,8 +27,8 @@ strip_comments()
     echo $@ | sed -e 's/\(.*\)#.*/\1/'
 }
 
-# Check if fence kdump is configured in cluster
-is_fence_kdump()
+# Check if fence kdump is configured in Pacemaker cluster
+is_pcs_fence_kdump()
 {
     # no pcs or fence_kdump_send executables installed?
     type -P pcs > /dev/null || return 1
diff --git a/kdumpctl b/kdumpctl
index 398858d..db673e4 100755
--- a/kdumpctl
+++ b/kdumpctl
@@ -192,7 +192,7 @@ function check_fence_kdump()
 	local image_time=$1
 	local cib_time
 
-	is_fence_kdump || return 1
+	is_pcs_fence_kdump || return 1
 
 	cib_time=`pcs cluster cib | xmllint --xpath 'string(/cib/@cib-last-written)' - | \
 		  xargs -0 date +%s --date`
-- 
1.8.3.1

